examples/notebooks/beam-ml/mltransform_basic.ipynb

```diff
@@ -6,9 +6,6 @@
         "id": "view-in-github",
         "colab_type": "text"
       },
-      "source": [
-        "<a href=\"https://colab.research.google.com/github/AnandInguva/beam/blob/mltransform_notebook/examples/notebooks/beam-ml/mltransform_basic.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
-      ]
     },
```

Comment:
Oh, I think we need to remove the full cell (from line 3 to this line). That should also fix the broken formatting - https://github.com/apache/beam/blob/2a7c9ff27fdbe2c6ed98945dd3f13db6b59f44d1/examples/notebooks/beam-ml/mltransform_basic.ipynb

Reply:
Ah, okay, I wondered about that. I removed the full cell.